src/main/app/case/CosApiClient.ts

```diff
@@ -182,11 +182,10 @@ export class CosApiClient {
   }
 
   /**  submit respondent response*/
```

Comment:
relevance?

Reply:
The data sent from here to cos is never used in the backend
so removed this unwanted data transfer